packages/rich-text-editor/src/RichTextEditor/controlmap.ts

```diff
@@ -176,13 +208,15 @@ export function buildControlMap(
 
     toolbarControls[groupIndex].push(
       {
-        action: liftOrIndentList("lift"),
+        action: liftList(),
```

Comment:
I should have picked this up earlier, but putting a function here with parentheses will invoke the function right?

Reply:
I believe what it returns is a `Command` that is used to invoke `liftListItem` and shouldn't actually execute

Reply:
although it might be better to do something like this and update the indentList function to:
```
const toggleIndentList: Command = (
  state: EditorState,
  dispatch: ((tr: Transaction) => void) | undefined
) => {
  const { $from } = state.selection
  const listItemNode = $from.node($from.depth - 1)?.type

  return sinkListItem(listItemNode)(state, dispatch)
}
```

Reply:
Oh so it's actually invoking this function here on purpose, in order to get the function that it returns. Mind blown. I guess the 2 without parameters (`liftList` and `indentList`) could just directly be the functions they are returning in that case, rather than a function that returns a function?